tensorflow/python/ops/variable_scope_test.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2015 The TensorFlow Authors. All Rights Reserved.
```

Comment:
@rohan100jain So I don't need to add the test cases in `variable_scope_test.py`?